lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -35,7 +35,7 @@ capabilities:
   _exclude: [ios_xr]
   multiple:
   get_context: ["<name>"]
-  get_command: "show interface capabilities"
+  get_command: "show interface <name> capabilities"
```

Comment:
Why does this one use `<name>` vs `<show_name>`?

Reply:
This is actually a completely different command from the other `get_command` strings and it also overrides the `_template` `get_command` string. Since it's a different command there is no cache re-use. 
We use the capabilities method to help with minitest / beaker only, and otherwise it's not used.